spec/models/participation_record_spec.rb

```diff
@@ -1,58 +0,0 @@
-# frozen_string_literal: true
-
-require "rails_helper"
-require "aasm/rspec"
-
-RSpec.describe ParticipationRecord, type: :model do
```

Comment:
I don't see anything replacing this test.

Is that because we are getting rid of this functionality? If so... Do we have anything else using `aasm/rspec`?

Reply:
No. It's all been pulled out.